### PR TITLE
changes

### PR DESCRIPTION
**Pull request by blodvan1**
Made changes to the code so that AST no longer clones child objects

### CODE REVIEW

**Comment by bazzilic** [comment]
What is this?

**Comment by cheziyi** [comment]
@bazzilic For https://github.com/bazzilic/PrismaDB-INTERNAL-DEV/issues/107, I will take a look soon.

**Comment by bazzilic** [comment]
Right now looks as if simply all references to `Clone` were removed, but I might be wrong

File: PrismaDB-QueryAST/DDL/AlterTableQuery.cs [hunk 1]
```diff
@@ -27,16 +27,16 @@ public AlterTableQuery(AlterType type, string tableName)
...
             AlterType = type;
-            TableName = table.Clone();
+            TableName = table;
             AlteredColumns = new List<AlteredColumn>();
         }
 
         public AlterTableQuery(AlterTableQuery other)
         {
-            TableName = other.TableName.Clone();
+            TableName = other.TableName;
```
Comment (cheziyi) [inline PrismaDB-QueryAST/DDL/AlterTableQuery.cs]: As mentioned, in constructors that take another of the same object, it should clone.

Comment (blodvan1) [reply PrismaDB-QueryAST/DDL/AlterTableQuery.cs]: ok noted, will make the necessary changes

**Comment by cheziyi** [comment]
There are some instances such as https://github.com/blodvan1/PrismaDB-QueryAST/blob/master/PrismaDB-QueryAST/DML/Clauses/FromClause.cs#L206 that clones the child items, even though it is not a constructor that takes the same type

**Pull request merged**